CMakeLists.txt

```diff
@@ -54,25 +53,8 @@ if(MSVC)
     set(CMAKE_EXE_LINKER_FLAGS /MANIFEST:NO)
 endif()
 
-if(BUILD_EXTERNAL_GOOGLEBENCHMARK)
-    configure_file(modules/downloadGBenchmark.cmake.in googlebenchmark-download/CMakeLists.txt)
-    execute_process(COMMAND ${CMAKE_COMMAND} -G "${CMAKE_GENERATOR}" .
-                    RESULT_VARIABLE result
-                    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/googlebenchmark-download )
-    execute_process(COMMAND ${CMAKE_COMMAND} --build .
-                    RESULT_VARIABLE result
-                    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/googlebenchmark-download )
-
-    # Add googlebenchmark directly to our build. This defines
-    # the gtest and gtest_main targets.
-    add_subdirectory(${CMAKE_CURRENT_BINARY_DIR}/googlebenchmark-src
-                     ${CMAKE_CURRENT_BINARY_DIR}/googlebenchmark-build)
-    set(GBENCHMARK_INCLUDE_DIRS "${googlebenchmark_SOURCE_DIR}/include")
-    set(GBENCHMARK_LIBRARIES benchmark)
-else()
```

Comment:
Why removing this stuff?

Reply:
I thought conda is the only supported install method. if not, this must stay